simulator-spring-boot/src/test/java/org/citrusframework/simulator/service/impl/ScenarioActionServiceImplTest.java

```diff
@@ -159,8 +157,7 @@ private void verifyDtoPreparations() {
                 "scenarioActions",
                 "scenarioMessages")
             .hasFieldOrPropertyWithValue("executionId", expectedScenarioExecution.getExecutionId())
-            .hasFieldOrPropertyWithValue("scenarioName", expectedScenarioExecution.getScenarioName())
-            .hasFieldOrPropertyWithValue("status", UNKNOWN);
+            .hasFieldOrPropertyWithValue("scenarioName", expectedScenarioExecution.getScenarioName());
```

Comment:
Access by string should be avoided if there is no specific reason to do so.
```suggestion
        assertThat(capturedScenarioExecution.getScenarioName()).isEqualTo(expectedScenarioExecution.getScenarioName());
```

Reply:
imho a lack of assertJ, `extractingEquals(Function<T, R>, R)`.. because as soon as you extract something, you're stuck with that nested object. but I'll use getters instead.